src/Illuminate/Foundation/Console/ServeCommand.php

```diff
@@ -96,7 +96,7 @@ class ServeCommand extends Command
     #[\Override]
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
-        $this->phpServerWorkers = transform(env('PHP_CLI_SERVER_WORKERS', 1), function ($workers) {
+        $this->phpServerWorkers = transform((int) env('PHP_CLI_SERVER_WORKERS', 1), function (int $workers) {
             if (! is_int($workers) || $workers < 2) {
```

Comment:
```suggestion
            if ($workers < 2) {
```

You are now requiring that `$workers` is an int, therefor the `is_int` check can be removed.